### ISSUE TITLE
Verse block: lines are not rendered

### ISSUE THREAD

**Issue by katinthehatsite** [Automattic/wp-calypso#38411]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/block-editor/post/
2. Select the Verse block
3. Add poetry to the block
4. Publish the post

#### What I expected
 For the poem to be displayed on the site

#### What happened instead
The poem was rendered in one line:

<img width="920" alt="Screen Shot 2019-12-13 at 11 01 55 AM" src="https://user-images.githubusercontent.com/25575134/70813738-fec7de00-1d97-11ea-8b0c-63ec1174f66c.png">

#### Browser / OS version
Chrome: Version 78.0.3904.108 (Official Build) (64-bit)
OS: Catalina 10.15.1 (19B88)

#### Context / Source
The issue was reported by the user in 2559066-zen
As a workaround, I have suggested using the classic block and using preformatted option.
I also saw a similar report here: https://github.com/WordPress/gutenberg/issues/19083

I also tried testing on a Jurassic Ninja site and could not reproduce it.

**Comment by katinthehatsite** [comment]
I also tried testing on Jetpack Nighty and could not reproduce it:
<img width="1295" alt="Screen Shot 2019-12-13 at 11 32 35 AM" src="https://user-images.githubusercontent.com/25575134/70816008-96c7c680-1d9c-11ea-93b3-e085bf949bd9.png">

**Comment by lakellie** [comment]
Another report in 362602-hc and the user mentioned that if they edit the HTML and remove the `wp-block-preformatted` class, it fixes the issue.

**Comment by lakellie** [comment]
Another report in 2564769-zen

**Comment by jamiepalatnik** [comment]
Another report in 17383559-hc

**Comment by sophiegyo** [comment]
This was fixed in core WordPress: https://github.com/WordPress/gutenberg/pull/19173 

I tested on jurassic.ninja and it's fine in core/selfhosted (my test: https://d.pr/i/hKDN3C ), but evidently still an issue on .com!

Screenshot: https://d.pr/i/qzTp2m

Checked in two browsers:
Chrome 79.0.3945.88
Firefox 71.0

Report from: 2562683-zd

**Comment by catkit** [comment]
> This was fixed in core WordPress: WordPress/gutenberg#19173

@sophiegyo it doesn't mean it was released yet, it only means it was fixed and merged into the master branch.

When there's a new release then it will be distributed and fixed in wpcom as well.

**Comment by donalirl** [comment]
Seen again in #2585575-zen. Noting to myself to follow up with this user once a fix is released to WPcom.

**Comment by donalirl** [comment]
#2592777-zen

**Comment by donalirl** [comment]
Here's a CSS fix you can offer to users for now:

```
/* Verse Block: add the proper spacings */
pre.wp-block-verse {
    white-space: pre;
}
```

**Comment by thebud15** [comment]
#17595328-hc

**Comment by wpericam** [comment]
#17626709-hc

**Comment by lizswafford** [comment]
#799508-hc

**Comment by susanjanec** [comment]
#2391934-hc

**Comment by Agepeuve** [comment]
ES forums - https://es.forums.wordpress.com/topic/problemas-con-bloque-verso/
https://es.forums.wordpress.com/topic/bloque-de-versos-no-me-funciona/

**Comment by fresatomica** [comment]
Another report: https://en.forums.wordpress.com/topic/problem-with-block-verse/

**Comment by xpurichan** [comment]
Another report: 14895399-hc (moved to 2614817-zd).

**Comment by mmtr** [comment]
This should be solved next Monday once we activate Gutenberg 7.2.0 on WP.com: p7DVsv-7T1-p2

**Comment by lizswafford** [comment]
17822792-hc

**Comment by edequalsawesome** [comment]
#17824647-hc

**Comment by poojashetty18** [comment]
#2624629-zen

**Comment by mmtr** [comment]
This is fixed now that Gutenberg 7.2.0 is live on WP.com.
